docs/applications/add-application.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Add an Application
+intro: Add applications to your Devopness infrastructure environments so they can be managed and deployed manually or through automated workflows
```

Comment:
When we say "your Devopness infrastructure environments" doesn't it sound like their infrastructure environment is hosted by us?

A "manual deployment" can be understood as an "old fashioned" deployment (eg: ssh/sftp) and disassociating ourselves from that seems like a good idea to me.

A suggestion considering the points mentioned, feel free to adapt:
```suggestion
intro: Add applications to your infrastructure environments so they can be managed and deployed directly by web interface or through automated workflows
```

Reply:
Nice catch! Tks!